helpdesk_mgmt_timesheet/demo/helpdesk_mgmt_timesheet_demo.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
```

Comment:
There are a lot of changes due to `<data noupdate="1">` replacement in first migration commit that could be rollbacked IMO, in order to only make those changes that are strictly required

Reply:
This is because the linter now force to be only one level (not `<odoo>` and then `<data>`).